### PR TITLE
Support standing order,Future Dated, One Step Payment, Cross Border

### PR DESCRIPTION
**Pull request by mahendra-patel339**

### CODE REVIEW

**Review by JugrajSripalJain — COMMENTED** [review]
Please check the java sample for one-step comments and apply the same here pls.

**Comment by mahendra-aps** [comment]
> Please check the java sample for one-step comments and apply the same here pls.

done

File: Controllers/ApplicationController.cs [hunk 1]
```diff
@@ -144,20 +433,29 @@ public Task<string> RedeemPopup()
...
+                  // Create a token request to be stored
+                  TokenRequest.StandingOrderRequestBuilder(
+                            amount,
+                            currency,
+                            "MNTH",
+                            startDate.ToString("yyyy-MM-dd"),
+                            endDate.ToString("yyyy-MM-dd"),
+                            new List<TransferDestination> { destination })
+                        .SetDescription(description)
+                        .AddDestination(destination)
```
Comment (JugrajSripalJain) [inline Controllers/ApplicationController.cs:450]: The TokenRequest already sets the destination with StandingOrderRequestBuilder, do we need to addDestination again?

**Review by JugrajSripalJain — CHANGES_REQUESTED** [review]

File: Controllers/ApplicationController.cs (hunk 1 above)
Comment (mahendra-aps) [reply Controllers/ApplicationController.cs]: Fixed. Done

**Review by JugrajSripalJain — APPROVED** [review]

**Pull request merged**